spec/order_processor_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'order_processor'
+
+describe OrderProcessor do 
+
+	subject { OrderProcessor.new notifier, calculable }
+	let( :notifier ) { double 'notifier', :call => nil }
+
+	describe "#process" do
+		
+		context "when is accepted" do
+			it "pass call to notifier" do
+				expect( notifier ).to receive :call
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space inside parentheses detected.<br>Tab detected.